tests/components/grid_manipulation.test.ts

```diff
@@ -95,6 +102,24 @@ describe("Context Menu add/remove row/col", () => {
     });
   });
 
+  test("cannot delete nor hide all cols with contextmenu", async () => {
+    setSelection(model, [zoneToXc(model.getters.getSheetZone(model.getters.getActiveSheetId()))]);
+    simulateContextMenu(".o-col-resizer", COLUMN_D);
+    await nextTick();
+    expect(fixture.querySelector(".o-menu div[data-name='delete_column']")).toBeFalsy();
```

Comment:
it's more correct to check `toBeNull'

Reply:
Changed it :) We do it at like 876 places tho, but probably not worth changing it